cypress/integration/models/developer/applicationinventory/assessment.ts

```diff
@@ -221,10 +222,7 @@ export class Assessment extends Application {
     }
 
     click_assess_button(): void {
-        cy.waitForReact();
-        cy.react("p", { props: { "aria-label": "assess-application" } })
-            .should("be.enabled")
-            .click();
+        click(assessAppButton);
```

Comment:
Do we need this to be in a separate method ? It's just a one liner

Reply:
I wanted to do as little changes as possible, but I can fix it if you want.
The thing is that I like to leave `actions` as separate methods and click them when needed separately.

Reply:
Just checked that if I will remove this method something else will be affected, this method is called in other places directly. Will be risky to remove it.